web/package.json

```diff
@@ -10,6 +10,8 @@
     "@types/node": "^16.18.12",
     "@types/react": "^18.0.28",
     "@types/react-dom": "^18.0.11",
+    "pace": "^0.0.4",
```

Comment:
What is this `pace` library for?

Reply:
pace is for progress bar in cmd. We don't use it, but I added it by mistake. So I deleted it.